collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -292,6 +296,13 @@ public void onCreate() {
         }
 
         setupLeakCanary();
+
+        try {
+            JobManager.create(this).addJobCreator(new SmsSenderJobCreator());
```

Comment:
There already is a job creator `ServerPollingJobCreator`, so `SmsSenderJobCreater` can be merged with that and renamed to something like `BackgroundJobCreator`. What do you think?

Reply:
Yes, was planning to merge this into that one as soon as I updated the branch, which I did. Thanks for the reminder!